test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -397,6 +397,21 @@ class test_uct_cm_sockaddr : public uct_test {
         m_listen_addr  = GetParam()->listen_sock_addr;
         m_connect_addr = GetParam()->connect_sock_addr;
 
+        const ucs::sock_addr_storage &src_sock_addr =
+                                                GetParam()->source_sock_addr;
+        if (src_sock_addr.get_sock_addr_ptr() != NULL) {
+            char src_addr_cstr[UCS_SOCKADDR_STRING_LEN];
+            int sa_family   = src_sock_addr.get_sock_addr_ptr()->sa_family;
+            const char *ret = inet_ntop(sa_family,
+                                        src_sock_addr.get_sock_addr_in_buf(),
+                                        src_addr_cstr, UCS_SOCKADDR_STRING_LEN);
```

Comment:
can we use ucs_sockaddr_str?

Reply:
`ucs_sockaddr_str` also adds port, `inet_ntop` simplifies the test

Reply:
so ucs_str_sockaddr does not parse port, but ucs_sockaddr_str adds port?
seems confusing..

Reply:
want to move back to uct/rdmacm code? :)

Reply:
rename, for example: ucs_sock_ipstr_to_sockaddr

Reply:
ok